setup.py

```diff
@@ -144,7 +143,7 @@ def py2exe_options():
         'psutil',  # windows binary code.google.com/p/psutil/downloads/list
         'psycogreen',
         'psycopg2 >= 2.2',
-        'pychart',  # not on pypi, use: pip install http://download.gna.org/pychart/PyChart-1.39.tar.gz
+        'pychart',
```

Comment:
Needs to be 'python-chart'. I suspect the same is true for the other occurrence of 'pychart' in the py2exe bit.